front/app/containers/Admin/projectFolders/containers/settings/ProjectFolderForm/index.tsx

```diff
@@ -433,21 +433,21 @@ const ProjectFolderForm = ({ mode, projectFolderId }: Props) => {
                   },
                 },
                 {
-                  onError: async (result) => {
-                    if (isNilOrError(result)) {
-                      setSubmitState('apiError');
-                    }
+                  onError: async () => {
+                    setSubmitState('apiError');
+                  },
+                  onSuccess: async () => {
+                    setSubmitState('success');
                   },
                 }
```

Comment:
Could we use mutateAsync here too? I suppose you changed it elsewhere because of improved code readability?

Reply:
In general, we prefer to use the sync version of mutate as a general rule. `mutateAsync` is more of an escape hatch when the component is expecting a promise and it would be too much of a refactor to change that.